### ISSUE TITLE
Questions of benchmark

### ISSUE THREAD

**Issue by depblack** [EndlessSora/DeeperForensics-1.0#12]
As the title shows , I have difficulty in reproducing the Results of XceptionNet Baseline .
I hope you could show me some ”Not private“ details of your experiments if you still remember them.   Or point out the errors in my own process.

Thank you anyway.

Our total process is shown as follows:
1) Using face detection method(MTCNN) to detect all frames in FF++_C23 videos, to get original face bounding box --【Boxes only from FF++_c23】；
 
2) With the scale (=1.3), enlarge the bounding box(also trying to be a rectangle box ); Then I use the boxes to extract faces in both FF++_C23 videos and DF1.0--end2end--the corresponding fake videos ;  --【1.3 faces from Both】；

2.5) Then we have two big folders,  each has 1,000 sub-folders of images (1000+1000 == 2000)

3) The the XceptionNet is trained from the two Folders (train:val: test is about 7: 1：2,  so about 0.7x2000 ==1400 sub-folders), and each video/sub-folder produce 270 frames at regular intervals(like frame_0, frame_2, ..., frame_538, if total frames is larger than 540)--【270 frames from each video】

4) The parameter of XceptionNet is 
  4.1) batch_size = 32 , epoches = 40
  4.2) optimizer_ft = optim.Adam(model.parameters(), lr=0.0002) #Other Default 
  4.3) exp_lr_scheduler = lr_scheduler.StepLR(optimizer_ft, step_size=2, gamma=0.9)
 4.4）Val is done after each epoch has trained 

 5) The test process  is done with all the images of the test sub-folders (about 0.2 x 2000 == 400)
6) If test on other dataset, like end_to_end_level_1, the test set is also like above(about 0.2 x 2000 == 400 sub-folders)

**Comment by depblack** [comment]
add to 4, the XceptionNet also loads pretrained weights from ImageNet,
learning_rate = 2e-4
Transforms=transforms.Compose([
        transforms.Resize((299, 299)),
        transforms.ToTensor(),
        transforms.Normalize([0.5, 0.5, 0.5], [0.5, 0.5, 0.5])
    ])
Like which in FF++_Baseline(github)
